c++/src/kj/compat/tls.c++

```diff
@@ -267,8 +268,9 @@ private:
           return readBuffer.whenReady().then(kj::mvCapture(func,
               [this](Func&& func) mutable { return sslCall(kj::fwd<Func>(func)); }));
         case SSL_ERROR_WANT_WRITE:
+          KJ_DBG("want write");
           return writeBuffer.whenReady().then(kj::mvCapture(func,
-              [this](Func&& func) mutable { return sslCall(kj::fwd<Func>(func)); }));
+              [this](Func&& func) mutable { KJ_DBG("got write"); return sslCall(kj::fwd<Func>(func)); }));
```

Comment:
Leftover KJ_DBG.

(I peeked at the rest of the PR, but didn't fully review since I lack some context. Nevertheless, 👍 FWIW.)

Reply:
Yikes, thanks for the catch. I forgot about that debug because it never actually triggers in local testing (since the write buffer never fills up).